src/WebJobs.Script/Config/HostJsonFileConfigurationSource.cs

```diff
@@ -215,7 +215,11 @@ internal JObject LoadHostConfig(string configFilePath)
                                             ScriptConstants.WorkFlowExtensionBundleId :
                                             ScriptConstants.DefaultExtensionBundleId;
 
-                        hostConfigObject = TryAddBundleConfiguration(hostConfigObject, bundleId);
+                        string bundleVersion = _configurationSource.Environment.IsLogicApp() ?
```

Comment:
Could you please add comments around this explaining that this is temporary until we support configuring host.json at app creation time (via ARM template / REST API at minimum)